content/reference/admin/private_locations/kubernetes/configuration/index.md

```diff
@@ -7,6 +7,14 @@ lastmod: 2023-10-13T08:10:39+00:00
 weight: 220341
 ---
 
+## Instance Specifications
+
+We recommend that you use for your own load generators pods with at least 4 cores. See CPU requests and limits configuration below.
+
+You might want to tune the `Xmx` JVM options to half of the memory request.
+See `jvm-options` configuration below.
+If you don't, the JVM will use a max heap size of 1/4th of the physical memory.
+
```

Comment:
Should we also mention something about isolating nodes used for load testing from those used to, for instance, run the app under test (ie use taints and tolerations if needed)?

Reply:
Very good idea

Reply:
Done